x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/screens/home/index.tsx

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
My preference is to remove `screens` directory and replace it with something like
`x-pack/.../sections/epm/home_page/index.tsx`
with the home page specific components nested in another directory like:
`x-pack/.../sections/epm/home_page/components/category_facets.tsx`

Following this, details page for a particular package would fall under something like:
`x-pack/.../sections/epm/details_page/`

But as this is just moving files around in the end, I don't feel super strongly about this :)

Reply:
I agree, but I would prefer to do this in a follow-up PR.